packages/sdk/src/job.ts

```diff
@@ -70,8 +70,13 @@ export class Job extends BaseApi {
    * you should check the job status before downloading
    * @param id job id
    */
-  downloadOutput(id: string): Promise<FileDownloadResp> {
-    return getFile(`${this.route}/${id}/output`);
+  async downloadOutput(id: string): Promise<FileDownloadResp> {
+    const resp = await getFile(`${this.route}/${id}/output`);
+    // header['content-disposition'] value looks like: 'attachment; filename="pipcook-output-u9fo9dlt.tar.gz"'
+    let filename = ContentDisposition.parse(resp.headers['content-disposition']).parameters['filename'];
+    filename = filename || 'output.tar.gz';
```

Comment:
How about merge the above 2 expressions?

Reply:
Updated.